### PR TITLE
enhance mysql connection

### PR DESCRIPTION
**Pull request by mikbry**
DONE #19 #20 #21

### CODE REVIEW

**Comment by mikbry** [comment]
Tests are not working since last PR

File: tests/database/mysql/mySqlDatabase.test.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ import dbCreate from "zoapp-core/database";
 
 import { descriptor, dbConfig } from "../../test-config";
 
+jest.setTimeout(50000);
```
Comment (willdurand) [inline tests/database/mysql/mySqlDatabase.test.js]: I dont think we should be doing this, but maybe you could explain what happens?

Comment (mikbry) [reply tests/database/mysql/mySqlDatabase.test.js]: time to time it could took some delay to init/access DB

Comment (willdurand) [reply tests/database/mysql/mySqlDatabase.test.js]: I see, but I dont think we should be doing this. This is not really a best practice.

Comment (mikbry) [reply tests/database/mysql/mySqlDatabase.test.js]: Ok i removed. I think all is ok now ;-)

**Comment by willdurand** [comment]
you should rebase but that looks ok

**Comment by mikbry** [comment]
Heu i made a rebase `git pull --rebase origin master`

**Comment by willdurand** [comment]
> Heu i made a rebase git pull --rebase origin master

and force pushed after?

**Comment by willdurand** [comment]
Ok, I have updated the PR, should be good to go if travis-ci agrees

**Pull request merged**